.github/workflows/release.yaml

```diff
@@ -37,22 +37,24 @@ jobs:
         uses: actions/setup-python@v4
         with:
           python-version: "3.11"
+      - name: Install Pipenv
+        run: |
+          python -m pip install pipenv
```

Comment:
We should probably use the `--system` flag here, as there's no need to use a virtual env on the VM.

Reply:
Indeed I want to try that. I guess the build inside the venv is the reason that I encounter gbome_keyring errors on ubuntu currently. Also I would like to get rid of the `--dev` deployment. It should actually be productive. I thought of removing pyinstaller from the dev dependencies and install it in the action manual for installs.

The reason for doing it this way currently is that releases broke with the `--system` flag in my first tries.

Reply:
Your comment above made me thinking, shouldn't we also use `sync` instead of `install` in CICD, and maybe even when setting up the local venv?

Reply:
funny thing... `pipenv sync` also creates a virtual environment, that's why it still needs `pipenv sync --system` 🤷

Reply:
Yes, in this regard, it works just like `install`. However, it doesn't update the pip or lock file. That's why I think it should be used in CI/CD, and definitely when releasing.
Sorry if I confused you, this wasn't meant as an answer to the venv issue.

See here: https://pipenv.pypa.io/en/latest/commands/